index.js

```diff
@@ -87,6 +87,20 @@ new PokemonGoMITM({port: 8081})
 app.use(bp.json());
 app.use(cors());
 
+function renamePokemon(mon) {
+  // don't rename pokemon that have already been renamed
+  if (mon.nickname[2] === '%') return null;
```

Comment:
I've also noticed that pokemon that are <10% also lose their sorting value. I'd need to add leading zeroes to prevent that from happening.